app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -1,48 +1,22 @@
-<div id="status_chart">
-  <%= job_invocation_chart(job_invocation) %>
-</div>
-
-<div class="col-md-6 infoblock">
-  <h4><%= _('Target hosts') %></h4>
-  <% if job_invocation.bookmark.present? %>
-    <%= _('Bookmark') %> <%= job_invocation.bookmark.name %><br>
-  <% else %>
-    <%= _('Manual selection') %>
-  <% end %>
-  <%= 'using ' + _(Targeting::TYPES[job_invocation.targeting.targeting_type]) %>
-  <pre><%= job_invocation.targeting.search_query %></pre>
+<% template_invocations = job_invocation.pattern_template_invocations %>
+<div class='row'>
+  <div class="col-xs-6 col-sm-6 col-md-6">
+    <%= render :partial => 'card_results', :job_invocation => job_invocation %>
+  </div>
 
-  <%= _('Evaluated at:') %> <%= job_invocation.targeting.resolved_at %><br>
-  <% if job_invocation.pattern_template_invocations.size > 1 %>
-    <% job_invocation.pattern_template_invocations.each do |template_invocation| %>
-      <%= host_counter template_invocation.template.provider.humanized_name, ForemanTasks::Task::DynflowTask.for_action(Actions::RemoteExecution::RunHostJob).for_resource(template_invocation).uniq.size %>
+  <div class='col-xs-12 col-sm-6 col-md-6'>
+  <%= render :partial => 'card_target_hosts', :job_invocation => job_invocation %>
+  <% template_invocations.each do |template_invocation| %>
+    <% if template_invocation.effective_user %>
```

Comment:
this and total host cards seems to take too much space, they could live next to each other

Reply:
I agree, however it's easier to manage when there's no effective user set. If I put them next to each other, when there's no effective user set it looks weird to have a card occupy half the size of the one above